nautobot/extras/urls.py

```diff
@@ -170,6 +170,21 @@
         kwargs={"model": ConfigContextSchema},
     ),
     # contacts
+    path(
+        "contact-associations/map-contact-from-location/",
```

Comment:
```suggestion
        "contact-associations/create-contact-from-location/",
```

Reply:
Are we able to make this one view with 3 different tabs?

Reply:
I like the idea but it's looking like a *lot* of refactoring of this PR to make that happen. Will continue to investigate.

Reply:
Basically we have two distinct actions that need to be supported by this (set of) view(s) and form(s):
- Create a ContactAssociation between an existing Location and an existing Contact/Team (then clean up data on the Location)
- Create a new Contact/Team, then create a ContactAssociation between it and an existing Location (then clean up data on the Location).